monkey/monkey_island/cc/app.py

```diff
@@ -72,36 +73,54 @@ def serve_home():
     return serve_static_file(HOME_FILE)
 
 
-def init_app_config(app, mongo_url, data_dir: Path):
+def setup_authentication(app, data_dir):
```

Comment:
```suggestion
def setup_flask_security_too(app, data_dir):
```

Reply:
This is leaking implementation details. What happens if we change the way we authenticate? I say we leave it as it is.

Reply:
This is not an interface so I'd prefer to leak implementation details. In our case "authentication" is just `flask_security_too` package, so it describes the method best. But it's a matter of preference, to me readability > potentially having to rename 1 method call, but it's minor, so I won't push it.